istio_api.bzl

```diff
@@ -101,6 +101,36 @@ gogo_proto_compile(
     with_grpc = False,
 )
 
+
+gogoslick_proto_library(
+    name = "mixer/v1/template",
+    importmap = {
+        "google/protobuf/descriptor.proto": "github.com/gogo/protobuf/protoc-gen-gogo/descriptor",
+    },
+    imports = [
+        "../../external/com_github_google_protobuf/src",
```

Comment:
`../..` doesn't seem to be bazel-kosher?

Reply:
It's not, but it's also how we've been doing it since ~January. We should talk w/ bazel team about better solution.

Reply:
Yeah.. Bazel stuff need to be revisited soon. But this is how it has been for some time now for us.

Reply:
Acknowledged. We need to revisit the bazel stuff soon.

Reply:
there is also an open bug in several repos for how to import the well-known protos in a better way. we could probably fix this with `bind`s and the like, but those are also not recommended.